src/SprykerEco/Yves/Heidelpay/Plugin/Router/HeidelpayRouteProviderPlugin.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\Heidelpay\Plugin\Router;
+
+use Spryker\Yves\Router\Plugin\RouteProvider\AbstractRouteProviderPlugin;
+use Spryker\Yves\Router\Route\RouteCollection;
```

Comment:
This requires adding spryker/router to composer dependencies.

Reply:
also removed silex dependency

Reply:
you've removed nothing related to Silex in the code, only deprecated.
What is the reason for dependency removal?

Reply:
brought silex back